ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -5455,3 +5455,87 @@ AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_RHHC_NONCANONICAL_1:
   - heating_water_valve_percentage_command
   - min_zone_air_relative_humidity_sensor
   - min_zone_air_relative_humidity_setpoint
+
+AHU_DFVSC_DSPM_ECOND_DTM_OA_RDC_RHM_RCO2M_CHWVM:
+  guid: "8ef1a26a-fe8f-468a-ba96-ffd0f987c043"
+  description: "AHU with discharge air/fan, chilled water valve operation, humidity monitoring with dual dampers monitoring and control ."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFVSC
+  - DSPM
+  - ECOND
+  - DTM
+  - OA
+  - RDC
+  - RTM
+  - RCO2M
+  - CHWVM
+  opt_uses:
+    - filter_alarm
+
+AHU_DFVSC_ECOND_HWDC_CHWPVM_RDC_RCO2M_RTM_UV:
+  guid: "005172cf-d29a-4c10-94d2-2a5fb6fad51f"
+  description: "AHU with discharge air/fan, UV, chilled water valve and return air temperature monitoring with dual dampers monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFVSC
+  - ECOND
+  - HWDC
+  - CHWPVM
+  - RDC
+  - RCO2M
+  - RTM
+  - UV
+  opt_uses:
+    - filter_alarm
+
+AHU_DFVSC_ECOND_HWDC_CHWPVM_RTM_UV:
+  guid: "f7579d91-707a-4e48-af46-9a86d8d31b94"
+  description: "AHU with discharge air/fan, chilled water valve and return air temperature monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFVSC
+  - ECOND
+  - HWDC
+  - CHWPVM
+  - RTM
+  - UV
+  opt_uses:
+    - filter_alarm
+
+AHU_DFSS_DFVSC_ECOND_CHWDC_ZTC_CO2M_UV:
+  guid: "2389a986-5591-422c-8a13-f31c262ecee7"
+  description: "AHU with discharge air/fan, chilled water valve and zone air temperature monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - DFVSC
+  - ECOND
+  - CHWPVM
```

Comment:
this does show if the CHW valve controls to a temperature setpoint, which I would expect

Reply:
I have changed CHWPVM to CHWDC